ask-sdk-core/tst/util/RequestEnvelopeUtil.spec.ts

```diff
@@ -87,6 +88,20 @@ describe('RequestEnvelopeUtils', () => {
         }).to.throw(`Expecting request type of IntentRequest but got LaunchRequest.`);
     });
 
+    it('should return the intent request object', () => {
+        const request = getRequest<IntentRequest>(intentRequestEnvelope)
```

Comment:
Missing comma

Reply:
Thanks.
I've fixed it :)
https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/pull/582/commits/ae0e324d8c3aa8b5c7ff63c7fa016f82724405e9

Reply:
BTW.

Probably, the `gulp tslint` does not referenced the tst dir.
So I made a new issue about the problem. -> tslint dose not check tst dir files #585

Thanks.